README.md

```diff
@@ -74,9 +74,8 @@ If you want to compile it from source, you will need Go 1.7 or later, and the [G
 3. Install dependencies using Glide:  
   `glide install`
 4. Compile the binary:  
-  - Linux/macOS/*nix: `go build -o akamai`
-  - Windows `go build -o akamai.exe`
-5. Move the binary (`akamai` or `akamai.exe`) it to your `PATH`
+  - Linux/macOS/*nix: `go install -o $GOPATH/bin/akamai .``
```

Comment:
`go install` doesn't support `-o`, and the step before is `go build`; should we perhaps just combine the steps?